docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,32 @@
+# Release 465 (7 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in EWKB files with the `ST_GeomFromBinary`
```

Comment:
Might want to spell out or link to what EWKB files are, this is a new concept to me.

Reply:
added a link